upgrade-tests/src/test/java/io/zeebe/test/RollingUpdateTest.java

```diff
@@ -199,7 +199,7 @@ private void deployProcess(final ZeebeClient client) {
         .newDeployCommand()
         .addWorkflowModel(PROCESS, "process.bpmn")
         .send()
-        .join(5, TimeUnit.SECONDS);
+        .join(10, TimeUnit.SECONDS);
```

Comment:
I guess this is unintended to be in this PR?

Reply:
This is intentional. After activating test coverage, this test failed much more often than before.

Reply:
Really? It slows down the overall build that much?

Reply:
No, I just doubled it to be on the safe side